src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -250,6 +250,13 @@ module sonic-device_metadata {
                                  created, regardless of the CONFIG_DB buffers configuration.";
                 }
 
+                leaf supporting_bulk_counter_groups {
+                    type string;
+                    description "This field contains a list of counter groups that support bulk operation.
+                                 It does not check whether bulk operations are supported for counter groups in the list
```

Comment:
Who does not check?

Reply:
sairedis. will update it.

Reply:
I think we can put it this way `The system does not check whether ...`. I do not want to mention sairedis since it's an internal module of sonic.
What do you think?

Reply:
In my opinion, 'This field contains a list of counter groups that support bulk operation' is sufficient.

Reply:
Fixed.